bin/dotd

```diff
@@ -348,39 +348,56 @@ def main
       should_i 'set DTP message to in progress' do
         DevelopersTopic.set_dtp "in progress (@#{@dotd_name})"
       end
-      GitHub.open_pull_request_in_browser(
-        base: 'production',
-        head: test_green_commit,
-        title: "DTP (#{test_green_commit})"
-      )
-
-      should_i 'DTL' do
-        # TODO(asher): Remove this wait step if it is truly unnecessary now.
-        wait_for 'Slack#levelbuilder to indicate they are ready for a deploy'
 
-        should_i 'commit content changes from levelbuilder, to help prevent merge conflicts' do
-          content_push 'levelbuilder-staging', 'levelbuilder'
+      # As GitHub's API (all of ours, Octokit's and github.com's) does not allow creating a pull
+      # request from a commit hash (only a branch name), we create and push a branch.
+      branch_name = "dtp_candidate_#{test_green_commit}"
+      begin
+        run_on(
+          'test',
+          "cd test && git checkout test && git checkout -b #{branch_name} #{test_green_commit} && git push origin #{branch_name}"
```

Comment:
Wouldn't that push the head of test to the remote branch? Here we want to push the current active branch, which this command will do AFAICT